### DISCUSSION TITLE
Manually se 'no data' value for raster

### DISCUSSION THREAD

**Discussion by geojed** [tylermorganwall/rayshader#172] [Q&A]
Hi,
Im having some difficulty rendering irregularly  (non rectangular) geotiff raster regions. The problem seems to arise from the nodata value being interpreted as elevation. Is there a way to manually set values to ignore when rendering elevations?

Thank you!

**Comment by tylermorganwall** [comment]
Make sure you have converted your elevation data to a matrix before passing it to rayshader.
